### PR TITLE
dependency-check 6.5.0

### PR DESCRIPTION
**Pull request by jeremylong**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by carlocab** [comment]
```
Error:  Unable to continue dependency-check analysis.
Error:  One or more fatal errors occurred
Error:  Unable to download meta file: https://nvd.nist.gov/feeds/json/cve/1.1/nvdcve-1.1-modified.meta
Error:  No documents exist
```

File: Formula/dependency-check.rb [hunk 1]
```diff
@@ -52,3 +52,4 @@ def install
     assert_predicate testpath/"dependency-check-report.xml", :exist?
   end
 end
+
```
Comment (carlocab) [inline Formula/dependency-check.rb:55]:
```suggestion
```

**Comment by jeremylong** [comment]
If this last run fails - I have one more option.  The issue is that the NVD has implemented fairly strict rate limiting. This does not affect normal operation of dependency-check - but if you kick off multiple fresh installs from likely the same IP the NVD will end up blocking some of dependency-check downloads. Dependency-check normally caches the results so multiple executions is not an issue - it is only when we have multiple completely fresh installs like we do with the homebrew build...  If this fails again I have another option I will implement in the morning.

**Review by carlocab — APPROVED** [review]
Looks like this worked. Thanks @jeremylong!

**Pull request closed without merging**